src/vs/base/parts/tree/browser/treeView.ts

```diff
@@ -444,7 +444,7 @@ export class TreeView extends HeightMap {
 
 		this.domNode = document.createElement('div');
 		this.domNode.className = `monaco-tree no-focused-item monaco-tree-instance-${this.instance}`;
-		this.domNode.tabIndex = 0;
+		this.domNode.tabIndex = context.options.preventRootFocus ? -1 : 0;
```

Comment:
Is preventRootFocus still a good name for the option? And were you able to figure out why we need the tabIndex for quick navigate?